RecoParticleFlow/PFClusterProducer/interface/PFRecHitQTests.h

```diff
@@ -259,8 +259,9 @@ class PFRecHitQTestHCALThresholdVsDepth : public PFRecHitQTestBase {
     {
       std::vector<edm::ParameterSet> psets = iConfig.getParameter<std::vector<edm::ParameterSet> >("cuts");
       for (auto & pset : psets) {
-        depths_.push_back(pset.getParameter<int>("depth"));
-        thresholds_.push_back(pset.getParameter<double>("threshold"));
+        depths_=pset.getParameter<std::vector<int> >("depth");
+        thresholds_=pset.getParameter<std::vector<double> >("threshold");
```

Comment:
here the size of depths_ and thresholds_ vectors should be the same.
Please add a check here as well.